pkg/controller/sshd/sshd_controller.go

```diff
@@ -0,0 +1,564 @@
+		}
+	} else {
+		// Deployment exists, check if it's updated.
+		if !reflect.DeepEqual(foundDeployment.Spec, deployment.Spec) {
+			// Specs aren't equal, update and fix.
+			r.SetSSHDStatusPending(instance, reqLogger, "Updating deployment", "from", foundDeployment.Spec, "to", deployment.Spec)
+			foundDeployment.Spec = *deployment.Spec.DeepCopy()
+			if err = r.client.Update(context.TODO(), foundDeployment); err != nil {
+				r.SetSSHDStatusError(instance, reqLogger, "Failed to update deployment", err)
+				return reconcile.Result{}, err
+			}
+		}
+	}
+
+	// Install Service
+	foundService := &corev1.Service{}
+	service := newSSHDService(instance)
+	serviceName, err := client.ObjectKeyFromObject(service)
+	if err != nil {
+		return reconcile.Result{}, err
+	}
+	if err = r.client.Get(context.TODO(), serviceName, foundService); err != nil {
+		if errors.IsNotFound(err) {
+			// Create a new Service.
+			r.SetSSHDStatusPending(instance, reqLogger, "Creating service")
+			if err = controllerutil.SetControllerReference(instance, service, r.scheme); err != nil {
+				r.SetSSHDStatusError(instance, reqLogger, "Failed to set service controller reference", err)
+				return reconcile.Result{}, err
+			}
+			if err = r.client.Create(context.TODO(), service); err != nil {
+				if errors.IsAlreadyExists(err) {
+					return reconcile.Result{Requeue: true}, nil
+				}
+				r.SetSSHDStatusError(instance, reqLogger, "Failed to create service", err)
+				return reconcile.Result{}, err
+			}
+			// Reconcile again to get the new Service and give AWS time to create the ELB.
+			reqLogger.Info("Service was just created, so let's try to requeue to set it up")
+			return reconcile.Result{Requeue: true, RequeueAfter: 10 * time.Second}, nil
+		} else {
+			return reconcile.Result{}, err
+		}
+	} else {
+		// Service exists, check if it's updated.
+		// XXX Copy system-assigned fields to satisfy reflect.DeepEqual.
+		copiedService := service.DeepCopy()
+		copiedService.Spec.Ports[0].NodePort = foundService.Spec.Ports[0].NodePort
+		copiedService.Spec.ClusterIP = foundService.Spec.ClusterIP
+		copiedService.Spec.HealthCheckNodePort = foundService.Spec.HealthCheckNodePort
+		if !reflect.DeepEqual(foundService.Spec, copiedService.Spec) {
+			// Specs aren't equal, update and fix.
+			r.SetSSHDStatusPending(instance, reqLogger, "Updating service", "from", foundService.Spec, "to", service.Spec)
+			foundService.Spec = *copiedService.Spec.DeepCopy()
+			if err = r.client.Update(context.TODO(), foundService); err != nil {
+				r.SetSSHDStatusError(instance, reqLogger, "Failed to update service", err)
+				return reconcile.Result{}, err
+			}
+			// Requeue to give AWS time to apply the changes.
+			reqLogger.Info("Requeuing after service update")
+			return reconcile.Result{Requeue: true, RequeueAfter: 10 * time.Second}, nil
+		}
+	}
+
+	// Create a Route53 DNS entry for the service's load balancer.
+	// FIXME Consider using https://github.com/openshift/external-dns
```

Comment:
```suggestion
	// TODO Consider using https://github.com/openshift/external-dns
```

Minor nit - Not really a FIXME, more of a to do item, and 👍  for using it when it comes around.